strings/phobia.json

```diff
@@ -213,5 +213,59 @@
 		"as (unknown)",
 		"unknown",
 		"stranger danger"
+	],
+
+"birds": [
+		"when are we adding vox",
+		"when you send me the sprites",
+		"or just link me to whichever server repo has them",
+		"i will port them right now",
+		"birdemic",
+		"bird",
+		"beak",
+		"poly",
+		"wing",
+		"wings",
+		"claw",
+		"peck",
+		"pecker",
+		"cracker",
+		"kiki yaya",
+		"ki ki ya ya",
+		"vox",
+		"flaps their wings",
+		"flaps his wings",
+		"flaps her wings",
+		"flaps their wings angrily",
+		"flaps his wings angrily",
+		"flaps her wings angrily",
+		"feather",
+		"feathered friend",
+		"avian",
+		"aviary",
+		"owl",
+		"owlman",
+		"griffin",
+		"kakaw",
+		"ka kaw",
+		"eagle",
+		"chick",
+		"chicken",
+		"fowl"
+	],
+
+"falling": [
+		"hold on",
+		"hang in there",
+		"chasm",
+		"pit",
+		"hole",
+		"fall",
+		"falling",
+		"fell",
+		"let go",
+		"i'll catch you",
+		"slip",
+		"slipped"
```

Comment:
add slipping

Reply:
and slips

Reply:
"slip" will catch both

Reply:
oh ok good

why is "slipped" separate then though?

Reply:
How does the check work?

I know it's findtext() but I don't quite understand if that means it cares for words and whitespace. DMref doesn't say. Mainly because elsewhere in this file there are examples of " hos ", "hos" and "rd" which sort of makes me wonder what happens if I want to say "Awa*rd* the greyshirt a skateboa*rd* for winning the thunde*rd*rome with a ha*rd* glass sha*rd*" and if the conspiracy one lets you say "CE" because you're not saying " CE "

Reply:
It _will_ detect the strings even in the middle of other words, and yeah, authority phobia will detect these words when it shouldn't. If you feel like it you could put spaces on the sides of those and prevent a bug report when someone bothers to put a tinfoil hat on a month from now.

Reply:
I'll fix my ones and slot that in a different PR.

It sounds more complex than just slapping spaces on the sides given that'd miss out on punctuation.